src/man/firejail-profile.txt

```diff
@@ -269,9 +278,13 @@ blacklist-nolog /usr/bin/gcc*
 .TP
 \fBbind directory1,directory2
 Mount-bind directory1 on top of directory2. This option is only available when running as root.
+Directories will retain the ownership and permissions of the original directory being mounted over. (directory2)
+After termination, modificationss affect the overlay directory. (directory1)
 .TP
 \fBbind file1,file2
 Mount-bind file1 on top of file2. This option is only available when running as root.
+Files will retain the ownership and permissions of the original file being mounted over (file2)
```

Comment:
Suggestion:
Files will retain the ownership and permissions of the original file being mounted over (file2).
After termination, deletes do not persist but writes affect the overlayed file (file1).